lib/jsdp-value-converter.js

```diff
@@ -0,0 +1,94 @@
+var JuttleMoment = require("juttle/lib/moment").JuttleMoment;
+var moment = require("moment");
+var _ = require("underscore");
+
+var dateConverter = {
+    isConvertibleToJSDP: function(value) {
+        return value instanceof JuttleMoment && value.moment;
+    },
+    isConvertibleToJuttle: function(value) {
```

Comment:
Yea, I had somewhat of a rough time with the naming. Will rename to `xxxJSDPValue` and `xxxJuttleValue`.